tests/chart_tests/test_prometheus_config_configmap.py

```diff
@@ -51,11 +51,9 @@ def test_prometheus_config_configmap(self, kube_version):
         assert doc["metadata"]["name"] == "release-name-prometheus-config"
 
         config_yaml = yaml.safe_load(doc["data"]["config"])
-        assert [
-            x["tls_config"]["insecure_skip_verify"]
-            for x in list(config_yaml["scrape_configs"])
-            if x["job_name"] == "kubernetes-apiservers"
-        ] == [False]
+        houston_jobs = [x for x in list(config_yaml["scrape_configs"]) if x["job_name"] == "houston-api"]
```

Comment:
`config_yaml["scrape_configs"]` is already a list
```suggestion
        houston_jobs = [x for x in config_yaml["scrape_configs"] if x["job_name"] == "houston-api"]
```